ui/StatusQ/src/StatusQ/Components/StatusPageIndicator.qml

```diff
@@ -118,7 +118,6 @@ Item {
         model: d.pages
         delegate: StatusBaseButton {
             objectName: "Page-%1".arg(itemText)
-            width: d.buttonWidth
```

Comment:
This simply wasn't sufficient for large values like "100"; better not hardcode the UI element sizes